tests/test_feature_simple_cur.py

```diff
@@ -4,12 +4,13 @@
 from sklearn import exceptions
 
 from skmatter.datasets import load_csd_1000r as load
-from skmatter.feature_selection import CUR
+from skmatter.feature_selection import CUR, FPS
 
 
 class TestCUR(unittest.TestCase):
     def setUp(self):
         self.X, _ = load(return_X_y=True)
+        self.X = FPS(n_to_select=10).fit(self.X).transform(self.X)
```

Comment:
why do we need to do this and not just take the first 10 points? can you maybe add a comment

Reply:
Okay, I guess it is not really one of the bottlenecks of the tests. If it becomes we can change it and hardcode the indices.